backend/src/xfd_django/xfd_api/tasks/utils/cloudwatch_metrics.py

```diff
@@ -0,0 +1,92 @@
+"""Cloudwatch Metrics wrappers."""
+# Standard Python Libraries
+import functools
+import json
+import logging
+import time
+
+LOGGER = logging.getLogger(__name__)
+
+
+def emit_redshift_metric(query_name: str, duration_s: float, rows: int, success=True):
+    """
+    Emit timing metrics to CloudWatch via Embedded Metrics Format (EMF).
+
+    Appears automatically if logs go to CloudWatch.
+    """
+    payload = {
+        "_aws": {
+            "Timestamp": int(time.time() * 1000),
+            "CloudWatchMetrics": [
+                {
+                    "Namespace": "CyHy/Workers/Redshift",
+                    "Dimensions": [["QueryName", "Success"]],
+                    "Metrics": [
+                        {"Name": "DurationSeconds", "Unit": "Seconds"},
+                        {"Name": "RowCount", "Unit": "Count"},
+                    ],
+                }
+            ],
+        },
+        "QueryName": query_name,
+        "Success": str(success),
+        "DurationSeconds": duration_s,
+        "RowCount": rows,
+    }
+    # Print JSON to stdout so CloudWatch Logs automatically picks it up.
+    print(json.dumps(payload))  # noqa
+
+
+def emit_django_metric(stage: str, count: int, duration_s: float, success=True):
+    """Emit CloudWatch EMF metrics for Django ORM operations."""
+    payload = {
+        "_aws": {
+            "Timestamp": int(time.time() * 1000),
+            "CloudWatchMetrics": [
+                {
+                    "Namespace": "CyHy/Worker/Functions",
+                    "Dimensions": [["Stage", "Success"]],
+                    "Metrics": [
+                        {"Name": "DurationSeconds", "Unit": "Seconds"},
+                        {"Name": "RecordCount", "Unit": "Count"},
+                    ],
+                }
+            ],
+        },
+        "Stage": stage,
+        "Success": str(success),
+        "DurationSeconds": duration_s,
+        "RecordCount": count,
+    }
+    print(json.dumps(payload))  # noqa
```

Comment:
Consider using LOGGER for the opportunity for richer logging and resilience to failure. We can expand what we might include to the layload later if we choose. 

`LOGGER.info("EMF metric emitted", extra={"emf_payload": payload})
`